source/gui/addonGui.py

```diff
@@ -303,16 +305,19 @@ def getAddonStatus(self, addon):
 			# Translators: The status shown for a newly installed addon before NVDA is restarted.
 			statusList.append(_("Install"))
 		# in some cases an addon can be expected to be disabled after install, so we want "install" to take precedence here
-		elif globalVars.appArgs.disableAddons or addon.isDisabled:
+		# If add-ons are globally disabled, don't show this status.
+		elif addon.isDisabled and not globalVars.appArgs.disableAddons:

```

Comment:
Because you moved the `appArgs.disableAddons` condition out of `isDisabled`, we no longer have to guard against it here:
```suggestion
		elif addon.isDisabled:
```
Also remove the comment above.

Reply:
No we can't remove this guard or otherwise, if an add-on is disabled we will display "Disabled, Disabled after restart".

Reply:
ah, yes of course.